crit/cli/cli.go

```diff
@@ -242,14 +242,14 @@ var xCmd = &cobra.Command{
 		switch args[1] {
 		case "ps":
 			xData, err = c.ExplorePs()
-		case "fds":
+		case "fd":
 			xData, err = c.ExploreFds()
-		case "mems":
+		case "mem":
```

Comment:
To preserve compatibility with the Python version of `crit`, should we accept both `mems` and `mem`?